src/main/java/com/disi/TravelPoints/controller/LandmarkController.java

```diff
@@ -14,6 +14,8 @@
 @RestController
 @RequestMapping("api/v1/landmark")
 @RequiredArgsConstructor
+@CrossOrigin("http://localhost:3000")
```

Comment:
This annotation is redundant because in the "SecurityConfig" class there is code written for "cors configuration"